packages/node-cli/README.md

```diff
@@ -4,15 +4,29 @@ CLI for Node.js is an interactive tool that guides you through all the steps of
 
 ## Prerequisites
 The following software is required to work with SuiteCloud CLI for Node.js:
-- Node.js version 12.14 LTS
+- Node.js version 12.14.0 LTS or greater
 - Java JDK version 11
 
 Read the full list of prerequisites in [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_1558708810.html).
+
+## Supported Versions
+Although you may have access to older versions of CLI for Node.js, only the versions in this section are officially supported.
+
+Each version of CLI for Node.js in NPM works optimally with a specific NetSuite version. To check the equivalence among versions, see the following table:
+
+| Version in NPM | Version in NetSuite |
+|:--------------:|:-------------------:|
+| 1.0.X | 20.1 |
   
 ## Installation
-To install CLI for Node.js, run the following command:
+Since CLI for Node.js is a development tool, use a global instance to install it by running the following command:
+```
+npm install -g @oracle/netsuite-suitecloud-nodejs-cli
+```
+
+With NPM, you can simply use the following command:
 ```
-npm install @oracle/netsuite-suitecloud-nodejs-cli --global
+npx netsuite-suitecloud-nodejs-cli
```

Comment:
It should be `npx @oracle/netsuite-suitecloud-nodejs-cli ` but I wouldn't put it here. What npx does is:
pulls the package - installs it - run it - delete it; It can be used to test it, but each command takes about 5min to run